src/legacyilibassemble.js

```diff
@@ -183,8 +186,79 @@ function assemblejs() {
     }
 }
 
+function assembleCustomLocale(data) {
+    let allData = data;
```

Comment:
Merge with the other legacy ilib function, since they are mostly the same.

Reply:
I updated exist function to support this feature. Please check them again.